src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const employeeList = [...document.querySelectorAll('tr')].slice(1, -1);
+const mainList = document.querySelector('tbody');
```

Comment:
```suggestion
const tableBody = document.querySelector('tbody');
```
it is a more clear name :arrow_up:

Reply:
done